rust/agama-lib/share/examples/profile.jsonnet

```diff
@@ -1,4 +1,10 @@
+// This is a Jsonnet file. Please, check https://jsonnet.org/ for more
+// information about the language.
+
+// The hw.libsonnet file contains hardware information from the "lshw" tool.
```

Comment:
On  Leap 15.4 at least, lshw.rpm or jsonnet.rpm do not contain a hw.libsonnet file. Where do I find it or how do I know what API it provides?